src/main/resources/default/taglib/w/lookupValue.html.pasta

```diff
@@ -38,6 +38,7 @@
         new TokenAutocomplete({
             name: '@name',
             selectMode: SelectModes.SINGLE,
+            allowCustomEntries: false,
```

Comment:
shoudln't this be a parameter / bound to the LookupField?

Reply:
Yes i also think this should be read from the lookup field, as it has a constructor param controling this